repo/packages/M/minio/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "3.0",
+  "name": "minio",
+  "version": "0.0.1-RELEASE.2016-09-11T17-42-18Z",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/minio/minio.git",
+  "maintainer": "support@mesosphere.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server compatible with Amazon S3. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images.",
```

Comment:
Please add a link to `example` at the end of `description`
Something like: `Installation documentation: https://github.com/dcos/examples/tree/master/1.8/minio`

`Example` is basically an installation guide. Please create a PR against https://github.com/dcos/examples to add the example and please make sure to use the template from other examples. These examples finally show up in https://dcos.io/docs/1.8/usage/tutorials/